### ISSUE TITLE
instantiating abstract should give error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1554]
CREATE/NEW class which is abstract should give error

RAISE EXCEPTION TYPE abstract should give error

**Comment by larshp** [comment]
```abap
CLASS vehicle DEFINITION ABSTRACT.
ENDCLASS.
CLASS vehicle IMPLEMENTATION.
ENDCLASS.
DATA my_car TYPE REF TO vehicle.
CREATE OBJECT my_car TYPE vehicle.
```

**Comment by larshp** [comment]
ref https://github.com/abaplint/transpiler/issues/359

**Comment by larshp** [comment]
`NEW` cases missing
